### PR TITLE
OE0-65: Satisfy openIMIS Policy Holder Organization profile

### PR DESCRIPTION
**Pull request by malinowskikam**
Changes:
- PolicyHolderOrganisation converter fitted to the profile
- Added additional mapping and configurations for Organisation resource
- Added unit tests for PolicyHolderOrganisationConverter
- Added API GET test for Organisation

[https://openimis.atlassian.net/browse/OE0-65](https://openimis.atlassian.net/browse/OE0-65)

### CODE REVIEW

File: api_fhir_r4/mapping/organizationMapping.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            "display": "Union",
+        },
+    }
+
+
+class PolicyHolderOrganisationActivityMapping(object):
+    SYSTEM = urljoin(GeneralConfiguration.get_system_base_url(),
+                     R4OrganisationConfig.get_fhir_ph_organisation_activity_code_system())
+
+    fhir_ph_organisation_activity = {
```
Comment (dragos-dobre) [inline api_fhir_r4/mapping/organizationMapping.py:73]: This should be a CodeSystem and values should be taken from PH module. Please check with Severin for the values.

File: api_fhir_r4/mapping/organizationMapping.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    }
+
+    LEVEL_SYSTEM = f'{GeneralConfiguration.get_system_base_url()}/CodeSystem/organization-hf-level'
+
+
+class PolicyHolderOrganisationLegalFormMapping(object):
+    SYSTEM = urljoin(GeneralConfiguration.get_system_base_url(),
+                     R4OrganisationConfig.get_fhir_ph_organisation_legal_form_code_system())
+
+    fhir_ph_organisation_legal_form = {
```
Comment (dragos-dobre) [inline api_fhir_r4/mapping/organizationMapping.py:40]: This should be a CodeSystem and values should be taken from PH module. Please check with Severin for the values.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: api_fhir_r4/mapping/organizationMapping.py (hunk 1 above)
Comment (malinowskikam) [reply api_fhir_r4/mapping/organizationMapping.py:73]: We checked it with Seweryn and the values are only defined for frontend module, not backend

**Comment by malinowskikam** [comment]
@dragos-dobre I fixed the mappings to use policyholder module config to initialize.

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**